src/Raven.Client/Documents/Operations/Configuration/GetConflictSolverConfigurationOperation.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.ServerWide;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Configuration
+{
+    public sealed class GetConflictSolverConfigurationOperation : IMaintenanceOperation<ConflictSolver>
```

Comment:
move this to tests, at this point I do not see a reason to expose this

Reply:
done :)